tests/integration/cmd_link_unlink_test.go

```diff
@@ -133,5 +133,23 @@ var _ = Describe("odo link and unlink command tests", func() {
 
 			helper.CmdShouldPass("odo", "unlink", "backend", "--app", appName, "--port", "8080", "--context", frontendContext)
 		})
+
+		It("should successfully delete component after linked component is deleted", func() {
+			// first create the two components
+			helper.CopyExample(filepath.Join("source", "nodejs"), frontendContext)
+			helper.CmdShouldPass("odo", "create", "nodejs", "frontend", "--context", frontendContext, "--project", project)
+			helper.CmdShouldPass("odo", "push", "--context", frontendContext)
+			helper.CopyExample(filepath.Join("source", "nodejs"), backendContext)
```

Comment:
Maybe use some other component type like java?

Reply:
I used two nodejs components for tests to be faster. For the purpose of this test, it doesn't matter which components are started.